packages/ui/src/form/FormField/FormField.tsx

```diff
@@ -0,0 +1,65 @@
+import { FieldProps } from 'formik';
+import { kebabCase } from 'lodash';
+import React, { FunctionComponent, memo, ReactNode, useCallback } from 'react';
+
+import BasicFormField from '../BasicFormField/BasicFormField';
+import FormFieldError from '../FormFieldError/FormFieldError';
+import Label from '../Label/Label';
+
+export interface FormFieldProps {
+    additionalClassName?: string;
+    name: string;
+    label?: ReactNode | ((fieldName: string) => ReactNode);
+    labelContent?: ReactNode;
+    footer?: ReactNode;
+    id?: string;
+    input(field: FieldProps<string>): ReactNode;
+    onChange?(value: string): void;
+}
+
+const FormField: FunctionComponent<FormFieldProps> = ({
+    additionalClassName,
+    labelContent,
+    label,
+    onChange,
+    footer,
+    input,
+    name,
+    id,
+}) => {
+    const renderField = useCallback(
```

Comment:
I can tell we are missing floating label changes here.
Are we going to add this into the `index.ts` so that we can import `FormField` from here?